airflow/ui/src/layouts/Nav/UserSettingsButton.tsx

```diff
@@ -60,6 +63,9 @@ export const UserSettingsButton = () => {
           <FiClock size="1.25rem" style={{ marginRight: "8px" }} />
           {dayjs().tz(selectedTimezone).format("HH:mm z (Z)")}
         </Menu.Item>
+        <Menu.Item value="version">
+          Version <a href={`https://pypi.org/project/apache-airflow/${data?.version}/`} rel="noreferrer" style={{color: "#365f84"}} target="_blank">{data?.version}</a>
```

Comment:
Let's also the Link component from chakra-ui. See inside of DocsButton for an example. If you want a color you can just pass `color="fg.info` as a prop